### DISCUSSION TITLE
Serverinfo command (Inspired from Adonis)

### DISCUSSION THREAD

**Discussion by TheHackerLxr** [kohls-admin/kohls-admin#67] [Ideas]
![image](https://github.com/user-attachments/assets/fae84401-be4a-4058-84e7-9dff1b23a7f4)
![image](https://github.com/user-attachments/assets/4eabf2ab-4cbe-41be-8454-5efd63a659bb)
![image](https://github.com/user-attachments/assets/cf62c74d-f92d-42f7-b8a9-cff538cfb7e2)
A very useful feature when you need to see what the ping is on the server and other data, it is very useful (You dont need to add this). Thanks if anything!

**Comment by kohltastrophe** [comment]
This would be very useful to have!
